integration-tests/src/test/java/com/redhat/parodos/integration/notification/common/NotificationTestBuilder.java

```diff
@@ -0,0 +1,78 @@
+package com.redhat.parodos.integration.notification.common;
+
+import com.redhat.parodos.notification.sdk.api.ApiClient;
+import com.redhat.parodos.notification.sdk.api.ApiException;
+import com.redhat.parodos.notification.sdk.api.NotificationRecordApi;
+import com.redhat.parodos.notification.sdk.model.NotificationRecordResponseDTO;
+import com.redhat.parodos.notification.sdk.model.PageNotificationRecordResponseDTO;
+import com.redhat.parodos.sdkutils.NotificationServiceSdkUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class NotificationTestBuilder {
+
+	private static final String DEFAULT_PROJECT_NAME = "project-1";
+
+	private static final String DEFAULT_PROJECT_DESCRIPTION = "an example project";
+
+	private String workFlowName;
+
+	private ApiClient apiClient;
+
+	// private Consumer<WorkFlowDefinitionResponseDTO> workFlowDefinitionConsumer;
```

Comment:
Is there any reason to keep this commented code?

Reply:
No, deleted. thanks